templates/v3/includes/_field_dropdown.html

```
@@ -91,17 +92,18 @@
           x-show="!jsReady"
-          :disabled="jsReady"
+          :disabled="jsReady || {{disabled|default:"false"}}"
           @change="$dispatch('field-change', { name: '{{ name }}', value: $event.target.value, label: $event.target.options[$event.target.selectedIndex].text })">
     {% if placeholder %}<option value="">{{ placeholder }}</option>{% endif %}
     {% for value, label in options %}
       <option value="{{ value }}" {% if value == selected %}selected{% endif %}>{{ label }}</option>
     {% endfor %}
   </select>
-  <div class="dropdown" :class="{ 'dropdown--open': open{% if deselectable %}, 'dropdown--has-value': selected && selected !== defaultValue{% endif %} }" x-cloak>
+  <div class="dropdown{% if disabled %} dropdown--disabled{% endif %}" :class="{ 'dropdown--open': open{% if deselectable %}, 'dropdown--has-value': selected && selected !== defaultValue{% endif %} }" x-cloak>
     <div class="dropdown__trigger"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)